File: ApplicationClient/WebSimulation/Assets/Resources/Scripts/network/Handler/DeviceLayerHandler.cs
using UnityEngine;
public class DeviceLayerHanlder
{
    private static DeviceLayerHanlder instance;
    public static DeviceLayerHanlder GetInstance()
    {
        if (instance == null)
        {
            instance = new DeviceLayerHanlder();
        }
        return instance;
    }
    private DeviceLayerHanlder() { }

    public bool Handle(JSONObject jb)
    {
        string name = jb["name"].str;
        if (name == "imuphyportbroadcast")
        {
            return ImuPhyPortBroadCast(jb["code"].n, jb["id"].str);
        }
        else if (name == "dreadybroadcast") {
            return DReadyBroadcast(jb["code"].n, jb["id"].str);
        }
        else
        {
            return false;
        }
    }

    private bool ImuPhyPortBroadCast(float code,string id)
    {
        Debug.Log("ImuPhyPortBroadCast" + code);
        return false;
    }

    private bool DReadyBroadcast(float code, string id)
    {
        Debug.Log("DReadyBroadcast" + code);
        return false;
    }
}

File: RootServer/network/handler/devicelayerhandler.js
const netemailbaler = require('../netmailbaler')
class DeviceLayerHandler {
    constructor() {
        console.log("Init DeviceLayerHanlder");
    }
    handle(obj) {
        let name = obj.name;
        if (name == "setimuphyports") {
            return this.setimuphyports(obj.portsize);
        } else if (name == "imuphyportbroadcastresp") {
            return this.imuphyportbroadcastresp(obj.code);
        } else if (name == "dready") {
            return this.dready();
        } else if (name == "dreadybroadcastresp") {
            return this.dreadybroadcastresp(obj.code);
        } else {
            return false;
        }
    }
    setimuphyports(portsize) {
        console.log("setimuphyports " + portsize);
        netemailbaler.send(netemailbaler.imuphyportbroadcast(netemailbaler.netCodeRecorder.TOKENERR, "fufagnzhou"));
        netemailbaler.send(netemailbaler.dreadybroadcast(netemailbaler.netCodeRecorder.IPERR, "fufangzhou"));
        return false;
    }
    imuphyportbroadcastresp(code) {
        console.log("imuphyportbroadcastresp");
        return false;
    }
    dready() {
        console.log("dready");
        return false;
    }
    dreadybroadcastresp(code) {
        console.log("dreadybroadcastresp " + code);
        return false;
    }
}
module.exports = new DeviceLayerHandler();